sklearn/linear_model/tests/test_coordinate_descent.py

```diff
@@ -1632,24 +1630,18 @@ def test_enet_cv_sample_weight_sparse(estimator):
         reg.fit(X, y, sample_weight=sw)
 
 
-@pytest.mark.parametrize("backend", ["loky", "threading"])
 @pytest.mark.parametrize(
     "estimator", [ElasticNetCV, MultiTaskElasticNetCV, LassoCV, MultiTaskLassoCV]
 )
-def test_linear_models_cv_fit_for_all_backends(backend, estimator):
+def test_linear_models_cv_fit_readonly_data(estimator):
     # LinearModelsCV.fit performs inplace operations on input data which is
     # memmapped when using loky backend, causing an error due to unexpected
     # behavior of fancy indexing of read-only memmaps (cf. numpy#14132).
 
-    if parse_version(joblib.__version__) < parse_version("0.12") and backend == "loky":
-        pytest.skip("loky backend does not exist in joblib <0.12")
-
-    # Create a problem sufficiently large to cause memmapping (1MB).
     n_targets = 1 + (estimator in (MultiTaskElasticNetCV, MultiTaskLassoCV))
-    X, y = make_regression(20000, 10, n_targets=n_targets)
-
-    with joblib.parallel_backend(backend=backend):
-        estimator(n_jobs=2, cv=3).fit(X, y)
+    X, y = make_regression(1000, 10, n_targets=n_targets)
+    with TempMemmap(X) as X_readonly, TempMemmap(y) as y_readonly:
+        estimator(cv=3).fit(X_readonly, y_readonly)
```

Comment:
This is not testing what it's supposed to. You need to set `copy_X=False`

Reply:
thanks

Reply:
hum setting `copy_X=False` on the CV estimator itself will fail with `ValueError: output array is read-only`. I did to think deeper on the original problem.

Reply:
Makes sense. Asking copy_X=False on read-only array is kind of contradictory :)

Reply:
What I wanted originally was to be able to pass `max_n_bytes=0` to make it possible to use a small dataset, but it seems not possible with the `joblib.parallel_backend` context.

Reply:
> max_n_bytes=0 to make it possible to use a small dataset, but it seems not possible with the joblib.parallel_backend context.

Yes I wanted to do that as well. But the API is too limited.

I think the PR in its current state is enough as  non regression test for https://github.com/numpy/numpy/issues/14132.

Reply:
Actually this is wrong because we do force the copy in the meta-estimator and then the fancy indexing on the copy therefore does not trigger the workaround in scikit-learn. Let me close this PR and open another one with a different strategy.